sql/http_server.go

```diff
@@ -31,15 +30,7 @@ import (
 	"github.com/cockroachdb/cockroach/util"
 )
 
-var (
-	allowedEncodings     = []util.EncodingType{util.JSONEncoding, util.ProtoEncoding}
-	errNoDatabase        = errors.New("no database specified")
-	errNoTable           = errors.New("no table specified")
-	errEmptyDatabaseName = errors.New("empty database name")
-	errEmptyTableName    = errors.New("empty table name")
-	errEmptyIndexName    = errors.New("empty index name")
-	errEmptyColumnName   = errors.New("empty column name")
-)
+var allowedEncodings = []util.EncodingType{util.JSONEncoding, util.ProtoEncoding}
```

Comment:
Can you move these changes into a separate PR? They seem unrelated to the gRPC addition and worth doing independently.